parameter_check.h

```diff
@@ -0,0 +1,22 @@
+int check_Parametertype1(float min1, float max1, float Parameter_value1)
```

Comment:
Hard to recall what type1 and type2 are, right? Call them `checkAgainstMaxMinRange` and `checkAgainstMaxRange`